packages/cli/src/index.ts

```diff
@@ -61,6 +61,13 @@ class FileProcessor {
     this.processQueue();
   };
 
+  private onFileProcessingError = (err: any) => {
+    console.log(`Error processing file: ${err.stack || JSON.stringify(err)}`);
```

Comment:
I got the following error which does not seem to be  a node `Error` and would otherwise up as `#<Object>` without stringification:

```
{"type":"ServerError","bufferOffset":98,"severity":"ERROR","message":"syntax error at or near \")\""}
```

Reply:
Interesting, `ServerError` means that this error was thrown by Postgres server.
Looks like there is a syntax problem with the query (`syntax error at or near \`).
Do you have a reproducible example for this query?
You can also see more detailed logs by setting env `DEBUG=*` when running PgTyped.

Reply:
I was running the same query directly through postgres from my editor with no errors, I should be able to reproduce. I'll try and get a minimal case.